### PR TITLE
Update fibaro integration documentation

### PR DESCRIPTION
**Pull request by rappenze**
## Proposed change

Improve fibaro integration documentation with additional information of supported hubs and features.

Remove a chapter about using z-wave entities in automations, I think that is documented in general HA documentation and there is no sense to duplicate it here.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fibaro.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Fibaro
-description: Instructions on how to setup Fibaro Z-Wave hubs (HCL and HC2) and configure devices within Home Assistant.
+description: Instructions on how to setup Fibaro Home Center and Yubii Home within Home Assistant.
```
Comment (frenck) [inline source/_integrations/fibaro.markdown:3]: Should `Yubii Home` become a virtual integration instead?

File: source/_integrations/fibaro.markdown [hunk 2]
```diff
@@ -29,29 +29,29 @@ ha_config_flow: true
 ha_integration_type: hub
 ---
 
-The [Fibaro](https://fibaro.com/) hub is a controller mainly connecting to Z-Wave devices.
+The Fibaro integration allows you to connect Home Assistant to a Fibaro Home Center or a Yubii Home hub so that you can control and monitor the connected devices and run scenes. Home Assistant uses a local connection to connect to the Fibaro hub.
 
-There is currently support for the following device types within Home Assistant:
+# Supported hub models
+Fibaro Home Center 2, Home Center Lite, Home Center 3, Home Center 3 Lite and Yubii Home.
```
Comment (frenck) [inline source/_integrations/fibaro.markdown]:
```suggestion
## Supported hub models

Fibaro Home Center 2, Home Center Lite, Home Center 3, Home Center 3 Lite, and Yubii Home.
```

File: source/_integrations/fibaro.markdown [hunk 3]
```diff
@@ -29,29 +29,29 @@ ha_config_flow: true
...
 
+# Features
```
Comment (frenck) [inline source/_integrations/fibaro.markdown]:
```suggestion
## Features

```

File: source/_integrations/fibaro.markdown [hunk 4]
```diff
@@ -29,29 +29,29 @@ ha_config_flow: true
...
+1. Control devices connected to the hub and synchronize the state of the devices (see platforms for supported devices and capabilities).
+2. Entities are automatically added when configuring the Fibaro integration and upon restart of Home Assistant when changed in Fibaro hub.
```
Comment (frenck) [inline source/_integrations/fibaro.markdown]:
```suggestion
2. Entities are automatically added when configuring the Fibaro integration and upon restart of Home Assistant when changed in the Fibaro hub.
```

File: source/_integrations/fibaro.markdown [hunk 5]
```diff
@@ -29,29 +29,29 @@ ha_config_flow: true
...
+3. Support for multiple hubs.
+4. Activate scenes defined in the Fibaro hub.
+
+{% include integrations/config_flow.md %} 
+
+<div class='note'>
+It is recommended to assign a static IP address to your Fibaro controller. This ensures that it won't change its IP address, so you won't have to change the `url` if the controller reboots and comes up with a different IP address. See your router's manual for details on how to set this up. If you need the MAC address of your Fibaro, check the label on the bottom.
+</div>
```
Comment (frenck) [inline source/_integrations/fibaro.markdown]:
```suggestion
<div class='note'>

It is recommended to assign a static IP address to your Fibaro controller. This ensures that it won't change its IP address, so you won't have to change the `url` if the controller reboots and comes up with a different IP address. See your router's manual for details on how to set this up. If you need the MAC address of your Fibaro, check the label at the bottom.

</div>
```

Comment (frenck) [reply source/_integrations/fibaro.markdown]: Why do we have this note even? Why don't we detect updates of the IP address and reconnect automatically?

We should not have to burden users with these details if not needed.

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/fibaro.markdown (hunk 1 above)
Comment (rappenze) [reply source/_integrations/fibaro.markdown:3]: Yes probably we can add a virtual integration for Yubii.

But as I understand the integration documentation is not duplicated and just linked, so it makes probably sense to keep the description like this?

Also they state that I'm not allowed to use an official image from them when integrating the hubs with other software, so how should the images for the brand look like?
_Make sure it is very clear from all you do that your app belongs to you and not to FIBARO. Do not use any Home Center or FIBARO branding trademarks or trade dress in any logo or graphics._

Comment (frenck) [reply source/_integrations/fibaro.markdown:3]: That is sad to hear, but I think personally that is... typical...

File: source/_integrations/fibaro.markdown (hunk 5 above)
Comment (rappenze) [reply source/_integrations/fibaro.markdown]: The only thing Fibaro does is providing a mDNS entry. As I understand this is not enough to be discovered by HA.

Comment (frenck) [reply source/_integrations/fibaro.markdown]: Home Assistant supports discovery on mDNS (zeroconf), including updates.
Additionally, we can listen to DHCP requests from devices and updated config entries based on those responses.

That is of course, out of scope for this PR, but something that could be considered,  as there is no need to burden users with this.

**Review by frenck — APPROVED** [review]
Thanks, @rappenze 👍 

../Frenck

**Pull request merged**